lib/blob/blobstore.c

```diff
@@ -2659,11 +2659,6 @@ blob_free_cluster_cpl(void *cb_arg, int bserrno)
 {
 	struct spdk_blob_free_cluster_ctx *ctx = cb_arg;
 	spdk_bs_sequence_t *seq = ctx->seq;
-	struct spdk_blob *blob = ctx->blob;
-
-	if (blob != NULL) {
-		blob->active.num_allocated_clusters--;
```

Comment:
So in case of unmap operation the number of allocated clusters is not decremented?

Reply:
Good catch!
The upstream commits override my codes.

Reply:
Fixed. Thank you.